crates/codegen/parser_templates/src/rust/mod.rs

```diff
@@ -0,0 +1,6 @@
+pub mod cst;
```

Comment:
Not sure if it is a side effect of using less macros, but I noticed a major improvement in rebuild speeds before/after this PR (18s to 9s). Really exciting!